src/App.jsx

```diff
@@ -1,24 +1,15 @@
-import { Button, Row, TextView, contentView } from "tabris";
+import { Row, contentView } from "tabris"
 
 const App = {
-  start: ({ count, onIncrement, onDecrement }) => {
-    contentView.append(
-      <$>
-        <Row stretchY stretchX centerX centerY spacing={20} height={200}>
-          <Button center onSelect={onIncrement}>
-            Increment
-          </Button>
-          <TextView centerX centerY font="24px" text={count} />
-          <Button center onSelect={onDecrement}>
-            Decrement
-          </Button>
-        </Row>
-      </$>
-    );
-  },
-  setCount: ({ count }) => {
-    $(TextView).only().text = count;
+  init (attrs) {},
```

Comment:
I decided to use a familiar Elm-like pattern when building the widget controllers for elements like `Button` and `Text`.
* I'm not sure why `init` would be useful, but I imagine it is useful at some point.
* `update` is implemented with a mutation api, not ideal for some but it achieves the results
  * Although it possible to use an api like Immer to convert mutations to immutable changes
* In `render` I'm not sure I like having `handlers` as the third argument
  * It seems likely that it could be stored in `context`?
    * Maybe `context` should be renamed to something more familiar?
* I'm certain we should move the `contentView.append(view)` expression to outside `render`
  * This could move into the controller layer bindings where we define `UIElment`